planner/core/logical_plan_test.go

```diff
@@ -1640,6 +1640,39 @@ func (s *testPlanSuite) TestVisitInfo(c *C) {
 				{mysql.AllPrivMask, "test", "ttt", "", nil},
 			},
 		},
+		{
+			sql: "alter table t add column a int(4)",
```

Comment:
Please remember to update our document.

Reply:
https://github.com/pingcap/docs-cn/pull/1192#pullrequestreview-219277616